gulpfile.js

```diff
@@ -84,6 +84,11 @@ gulp.task('ps1tests', ['compileTests'], function (cb) {
         .pipe(gulp.dest(_testRoot));
 });
 
+gulp.task('copyTestRes', ['compileTests'], function (cb) {
```

Comment:
The previous task - ps1tests - only copies ps1 and json files so there is no way to use other kind of files for tests. I plan to add some unit tests that consume code analysis results which are xml files and .properties files so I think a general purpose mechanism would be good.